### PR TITLE
Remove unused ArenaParticipantsWorker related code

### PR DESCRIPTION
**Pull request by upa-r-upa**
Resolve https://github.com/planetarium/NineChronicles.Headless/issues/2579

Removed 2 files 
- [ArenaParticipantsWorker.cs](https://github.com/planetarium/NineChronicles.Headless/blob/2b3dfb4ec0d4ab2b4309213b12ac3713c9b151e5/NineChronicles.Headless/ArenaParticipantsWorker.cs) file.
- [ArenaParticipantsWorkerTest.cs](https://github.com/planetarium/NineChronicles.Headless/blob/2b3dfb4ec0d4ab2b4309213b12ac3713c9b151e5/NineChronicles.Headless.Tests/ArenaParticipantsWorkerTest.cs) file

### CODE REVIEW

**Comment by moreal** [comment]
Hello, @upa-r-upa. Thanks for your contribution 🙏🏻 But it seems that there are more work items to resolve the issue. Could you make your branch able to build?

**Comment by upa-r-upa** [comment]
Thanks! I've fixed it. 😸

**Comment by moreal** [comment]
It looks good to me! 😁  @ipdae Could you review this pull request?

File: NineChronicles.Headless.Executable/Program.cs [hunk 1]
```diff
@@ -476,11 +476,6 @@ IActionLoader MakeSingleActionLoader()
                                 })
                         );
 
-                    // worker
-                    if (arenaParticipantsSync)
```
Comment (ipdae) [inline NineChronicles.Headless.Executable/Program.cs:480]: Could you please delete these unused arguments?

Comment (moreal) [reply NineChronicles.Headless.Executable/Program.cs:480]: They can be resolved in the next PR.

**Review by moreal — APPROVED** [review]

**Comment by moreal** [comment]
Thank you for contributing! 🎉

File: NineChronicles.Headless.Executable/Program.cs (hunk 1 above)
Comment (Atralupus) [reply NineChronicles.Headless.Executable/Program.cs:480]: We can create new GFI yeah

Comment (moreal) [reply NineChronicles.Headless.Executable/Program.cs:480]: @Atralupus I'll open an issue before the end of the day.

Comment (moreal) [reply NineChronicles.Headless.Executable/Program.cs:480]: @Atralupus I opened it as https://github.com/planetarium/NineChronicles.Headless/issues/2598

**Pull request merged**